README.md

```diff
@@ -86,7 +278,7 @@ This is illustrated below.
 
 
 `````
-─── datagateway-api
+─── datagateway_api
```

Comment:
I've updated the directory structure in the README and I spent some time making the tree easy-to-generate in the future (I used to just make the changes manually...). It's quite a nice command because it uses the repo's `.gitignore` file to exclude log files etc. I'd say this is now resolved.